Pods/CocoaSoundCloudUI/Sources/SoundCloudUI/SCRecordingSaveViewController.m

```diff
@@ -798,17 +798,16 @@ - (UIView *)tableView:(UITableView *)aTableView viewForHeaderInSection:(NSIntege
     if (!text)
         return nil;
     
-    CGSize textSize = [text sizeWithFont:[UIFont systemFontOfSize:15.0]
-                       constrainedToSize:CGSizeMake(CGRectGetWidth(self.tableView.bounds) - 2 * [self cellMargin], CGFLOAT_MAX)
-                           lineBreakMode:UILineBreakModeWordWrap];
+//    CGSize textSize = [text sizeWithFont:[UIFont systemFontOfSize:15.0]
+//                       constrainedToSize:CGSizeMake(CGRectGetWidth(self.tableView.bounds) - 2 * [self cellMargin], CGFLOAT_MAX)
+//                           lineBreakMode:NSLineBreakByCharWrapping];
     
```

Comment:
Its in a view that we will never use in our app, but just commenting out for the sake of removing the warnings.